src/blockchains/__tests__/filecoin.test.ts

```diff
@@ -0,0 +1,65 @@
+import {LocalFilecoinProvider} from "@ukstv/local-filecoin-provider";
```

Comment:
Is this library available on github somewhere? I see it on npm and that it's based on the one from open work labs. Can it potentially be merged to their repo?

Reply:
It is on https://github.com/ukstv/local-filecoin-provider I believe it should eventually move to https://github.com/openworklabs/filecoin-wallet-provider indeed.